### PR TITLE
add example for running on multiple GPUs using Horovod

### PR DESCRIPTION
**Pull request by radekosmulski**

### CODE REVIEW

File: examples/02-Multi-GPU-Tensorflow-with-Horovod.ipynb [hunk 1]
```diff
@@ -0,0 +1,371 @@
+{
```
Comment (bschifferer) [inline examples/02-Multi-GPU-Tensorflow-with-Horovod.ipynb:1]: <p>Why is Horovod in brackets? [Horovod]? Do you want to add a link for it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by bschifferer** [comment]
The logo is the default one: http://developer.download.nvidia.com/compute/machine-learning/frameworks/nvidia_logo.png

**Review by bschifferer — CHANGES_REQUESTED** [review]
Can you update the logo with a specific link for the example?

Can you add a unittest? We do not likely have multi-GPU as an option. In the unittest on Merlin Models, I set the num_gpus = 1 for the unittest. (I am actually not sure, if it will be executed on CPU-only environments). Maybe we need another importskip for it

https://github.com/NVIDIA-Merlin/models/blob/main/tests/unit/tf/examples/test_usecase_data_parallel.py

File: examples/02-Multi-GPU-Tensorflow-with-Horovod.ipynb (hunk 1 above)
Comment (radekosmulski) [reply examples/02-Multi-GPU-Tensorflow-with-Horovod.ipynb:1]: <p>yes, apologies, that was the intention, fixed it now</p>

**Comment by radekosmulski** [comment]
rerun tests

**Review by bschifferer — APPROVED** [review]

**Comment by bschifferer** [comment]
rerun tests

**Comment by bschifferer** [comment]
rerun tests

**Pull request merged**